### PR TITLE
[SELS-TASK] Delete category page

### PR DESCRIPTION
**Pull request by kdacudag**
## Asana Link
https://app.asana.com/0/1202373230087288/1202373365645310/f

### Server
`cd server`
`php artisan serve --host localhost`

### Client
`cd client`
`npm start`

## Pre-conditions
Log in using admin credentials: `admin@gmail.com` `password`
Go to categories and delete a category

## Expected Output
A category will be deleted from the categories table.

## Screenshots
image

### CODE REVIEW

File: client/src/components/layouts/Navigation.js [hunk 1]
```diff
@@ -31,7 +32,7 @@ const Navigation = ({ users }) => {
     if (!auth.isSignedIn) {
```
Comment (drei001) [inline client/src/components/layouts/Navigation.js:32]: How about the checks if the user is not an admin?

Comment (kdacudag) [reply client/src/components/layouts/Navigation.js:32]: Will implement that sir

Comment (drei001) [reply client/src/components/layouts/Navigation.js:32]: Is this included in as sub-task or this will be a seperate PR?

Comment (kdacudag) [reply client/src/components/layouts/Navigation.js:32]: Is it fine to just create another task in Asana for a separate PR for this one sir?

Comment (drei001) [reply client/src/components/layouts/Navigation.js:32]: Ok. Create a separate task and PR. I'm ok with this, I will merge

**Pull request merged**